src/index.scss

```diff
@@ -533,7 +534,74 @@ PACKAGES
 		height: 330px;
 	}
 }
+#not-found-page {
+  padding: 20px;
+  background-color: $kpmp-gray;
 
+  .not-found-image {
+    max-width: 90%;
+    margin-top: 30px;
+    margin-right: 40px;
+  }
+
+  .not-found-container {
+    max-width: 60%;
+    margin: auto;
+  }
+
+  .not-found-text {
+    padding-top: 30px;
+
+    .not-found-regular {
+      font-size: 32px;
+    }
+
+    .not-found-small {
+      font-size: 24px;
+    }
+  }
+}
+@media (max-width: 764px) {
```

Comment:
I don't think we want this here, it looks like it's for the homepage, not the 404 page